library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -0,0 +1,33 @@
+package com.novoda.downloadmanager;
+
+import android.content.ContentResolver;
+
+import com.novoda.downloadmanager.lib.DownloadManager;
+
+public class DownloadManagerBuilder {
+
+    private ContentResolver contentResolver;
+    private boolean verboseLogging;
+
+    public static DownloadManagerBuilder create() {
+        return new DownloadManagerBuilder();
+    }
+
+    public DownloadManagerBuilder withVerboseLogging() {
+        this.verboseLogging = true;
+        return this;
+    }
+
+    public DownloadManagerBuilder with(ContentResolver contentResolver) {
```

Comment:
out of curiosity I always enforce things through the ctor, any reason for doing it through build instead?